_data/public-speaking.yml

```diff
@@ -1,3 +1,17 @@
+- name: "Devoxx"
+  by: "Simon"
+  event: "Devoxx"
+  date: 2024-10-03
+  summary: >
+    Devoxx 2024 was an incredible event, offering insightful 
+    experiences and a great chance to reconnect with familiar 
+    faces while meeting new ones.
+  content: "Code-Show"
+  level: Introduction
+  duration: "2h58min"
```

Comment:
30min


Also as per the guidelines: if it's "2h58" -> just make it 3h
Always round down/up so it's a nice round number